test/vms/cirros/Makefile

```diff
@@ -21,6 +21,8 @@ $(disk):
 	virt-customize --add $(work) \
 		--copy-in ../ramen:/tmp \
 		--run-command "/tmp/ramen/install" \
+		--run-command "sed -i 's/console=tty1/quiet/' /boot/grub/menu.lst" \
+		--run-command "sed -i 's/console=ttyS0//' /boot/grub/menu.lst" \
```

Comment:
Needs more work:
- split quiet option from other changes
  - removing console=tty seems unneeded
- test `console=ttyS0,115200` - default speed may be 9600, 12 times slower
- use ramen start log uptime to measure instead of ssh connection time